snapshots/overlay/overlayutils/check.go

```diff
@@ -114,6 +134,16 @@ func NeedsUserXAttr(d string) (bool, error) {
 		return false, nil
 	}
 
+	// userxattr not permitted on tmpfs https://man7.org/linux/man-pages/man5/tmpfs.5.html
+	isOnTempFs, err := IsPathOnTempFs(d)
+	if err != nil {
+		return false, err
```

Comment:
Better to just log the error and continue checking the kernel version

Reply:
Ok with which verbosity?